zebra-consensus/src/block/subsidy/funding_streams.rs

```diff
@@ -74,7 +74,7 @@ fn funding_stream_address_period(height: Height, network: Network) -> u32 {
     // - In Rust, "integer division rounds towards zero":
     //   https://doc.rust-lang.org/stable/reference/expressions/operator-expr.html#arithmetic-and-logical-binary-operators
     //   This is the same as `floor()`, because these numbers are all positive.
-    (height.0 + (POST_BLOSSOM_HALVING_INTERVAL.0) - (height_for_first_halving(network).0))
+    (height.0 + (POST_BLOSSOM_HALVING_INTERVAL as u32) - (height_for_first_halving(network).0))
```

Comment:
I'd prefer we use try_into() here, because `as` can do some unexpected conversions and wrapping.

Reply:
155ce35d1d